__tests__/app.test.js

```diff
@@ -41,4 +41,63 @@ describe("GET /api/topics", () => {
     .get("/api/topis")
     .expect(404)
   })
+})
+
+describe("GET /api/articles/article_id", () => {
+  test('200: returns an array of first article', () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(1);
+        articles.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+        })
+      })
+    })
+  })
+  test('200: returns an array of other viable articles', () => {
+    return request(app)
+    .get("/api/articles/12")
+    .expect(200)
+    .then(({body}) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(1);
+        articles.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+        })
+      })
+    })
+  })
+  test('404: returns a 404 error if path is incorrect', () => {
```

Comment:
This is a good 404 to test undefined paths. You should also test for an valid but non existent id on this endpoint eg `/api/article/9999` - this would also be best as a 404